src/main/telemetry/crsf.c

```diff
@@ -297,6 +302,60 @@ void crsfFrameBatterySensor(sbuf_t *dst)
     sbufWriteU8(dst, batteryRemainingPercentage);
 }
 
+#if defined(USE_BARO) && defined(USE_VARIO)
+// pack altitude in decimeters into a 16-bit value.
```

Comment:
We use altitude in cm (centi-meters) within Betaflight.
It would be better for altitude control like GPS Rescue if we could receive altitude values with cm steps.
Baro isn't much use for absolute altitude, especially with temperature changes and drift over time, and using Betaflight over 3000m above takeoff point is exceptionally uncommon.  
If you are limited to that range, perhaps use two data values, one to count height multiples above 3,276m?

Reply:
how is this relevant for telemetry (going _out_)? seems there the value is purely informational, where its resolution is less critical.